apps/storefront/src/types/products.ts

```diff
@@ -209,3 +209,25 @@ export interface CalculatedProducts {
   items: Partial<Calculateditems>[];
   currency_code: string;
 }
+
+export interface TaxDiscountAmountProps {
+  asEntered: number;
+  as_entered?: number;
+  enteredInclusive: boolean;
+  entered_inclusive?: boolean;
+  taxExclusive: number;
+  tax_exclusive?: number;
+  taxInclusive: number;
+  tax_inclusive?: number;
+}
+
+export interface BulkPriceItem {
+  minimum: number;
+  maximum: number;
+  discountAmount: number;
+  discount_amount?: number;
+  discountType: string;
+  discount_type?: string;
```

Comment:
Having two possible representation of each key is a bit awkward. This is normally better solved with an anti corruption layer, basically transforming all the data at the edge of the application so the interfaces are consistent inside the app.